src/pages/community-settings/violation-detection/index.tsx

```
@@ -1,277 +1,276 @@
+													<Chip
+														label={option}
+														{...getTagProps({ index })}
+														variant="outlined"
+														size="small"
+														sx={{
+															borderRadius: "4px",
+															borderColor: "#D1D1D1",
+															backgroundColor: "white",
+															color: "black",
+														}}
+													/>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)